variables.tf

```diff
@@ -53,4 +53,9 @@ variable "trail_bucket_default_encryption" {
 variable "trail_bucket_default_encryption_key" {
   description = "Defines if the KMS encryption key, used if trail_bucket_default_encryption is set to aws:kms"
   default     = ""
-}
\ No newline at end of file
+}
+
```

Comment:
Can you please add a var for the Key-Name? It is ok if it defaults to "Name", but we for example use lowercase keys which could configured then.